src/Dialog/index.scss

```diff
@@ -66,6 +71,7 @@
   border-top-left-radius: var(--border-radius-m);
 
   max-height: 96%; // relative to parent, .nds-dialog-focuslock
+  min-width: 100dvw;
```

Comment:
This fixes an unrelated minor bug I found when testing. It's quite an edge case, but if a Dialog doesn't have very much content in it (too little to wrap a single line of text), it won't take up the whole screen width at mobile sizes.

This is a safe change because we already override `min-width` for desktop sizes.